src/InlineError/InlineError.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { dcnb } from "cnbuilder";
+import Icon from "react-hero-icon";
+
+export const InlineError = ({ className, text, icon, iconType, isError }) => {
+  const iconProps = { height: 20, width: 20 };
+
+  // Default Icon.
+  let defaultIcon = (
+    <Icon
+      icon="x"
+      type="sold"
+      aria-hidden="true"
+      className="su-text-digital-red su-inline su-mr-3"
+      {...iconProps}
+    />
+  );
+
+  if (icon) {
+    defaultIcon = (
+      <Icon
+        icon={icon}
+        type={iconType}
+        aria-hidden="true"
+        className="su-text-digital-red su-inline su-mr-3"
+        {...iconProps}
+      />
+    );
+  }
+
+  return (
+    <>
+      {isError && (
+        <div className={dcnb("su-text-digital-red", className)}>
+          <p className="su-text-16">
+            {defaultIcon}
+            {text}
+          </p>
+        </div>
+      )}
+    </>
+  );
+};
+
+// Prop Types.
+// -----------------------------------------------------------------------------
+InlineError.propTypes = {
+  // CSS Classes.
+  className: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.array,
+    PropTypes.object,
+  ]),
```

Comment:
I know this something that's been kind of standardized on here, but I feel like extending the `className` prop to anything other than it's native `string` value is a dangerous practice. Plus, I've already ran into it [throwing console warnings](https://github.com/SU-SWS/saa_alumni/pull/159#discussion_r705602877) in the homesite project.

Reply:
> I feel like extending the className prop to anything other than it's native string value is a dangerous practice

The trade here is the ability to unset styles that we have added to the outermost wrapper by passing in an object of falsy values for the things we don't want through to dcnb(). This can allow us to keep the list of outputted classnames cleaner instead of appending opposing classnames to the end of the class attribute.

eg: If we put `su-rs-4` but my project wanted no spacing we could pass in `{su-rs-4: false}` to remove it from the output instead of adding `su-rs-0`.